code/Driver/DriverExample.cpp

```diff
@@ -0,0 +1 @@
+
```

Comment:
Sorry for the late response, the Driver module is from the se-research/CaroloCup repo it's the module structure of the 2016 year CaroloCup, it handles parking and obstacle-avoidance.